solr/core/src/test/org/apache/solr/security/MultiAuthPluginTest.java

```diff
@@ -105,6 +110,10 @@ public void testMultiAuthEditAPI() throws Exception {
           new SecurityConfHandler.SecurityConfig()
               .setData(Utils.fromJSONString(multiAuthPluginSecurityJson)));
       securityConfHandler.securityConfEdited();
+
+      // verify "WWW-Authenticate" headers are returned
+      testWWWAuthenticateHeaders(httpClient, baseUrl);
```

Comment:
should this test actually be starting with `verify`?  Instead of `test`?   like `verifySecurityStatus`??

Reply:
Yes I think that make sense as _testWWWAuthenticateHeaders_ is not a UnitTest method.